aspnet/conceptual-overview/aspnet.rst

```diff
@@ -104,8 +124,23 @@ ASP.NET Core uses a new configuration model for handling of simple name-value pa
 
 See :doc:`/fundamentals/configuration` for more details on the new configuration system and :doc:`/fundamentals/environments` for details on how to work with environments in ASP.NET Core.
 
+Build web UI and web APIs using MVC
+-----------------------------------
+
+- ASP.NET Core helps you create well-factored and testable web apps. See :doc:`/mvc/index` and :doc:`/testing/index`.
+- `Razor <http://www.asp.net/web-pages/overview/getting-started/introducing-razor-syntax-c>`__ provides a productive language to create :doc:`Views </mvc/views/index>`
+- :doc:`Tag Helpers </mvc/views/tag-helpers/intro>` enable server-side code to participate in creating and rendering HTML elements in Razor files. Tag Helpers are one of the most popular new features of ASP.NET Core Mvc
```

Comment:
Remove "Tag Helpers are one of the most popular new features of ASP.NET Core Mvc".